app/models/search_tab.rb

```diff
@@ -57,16 +57,18 @@ def more_label(count)
   end
 
   # Provide the summary counts of record in the dropdown categories
-  # @Author: Jeffery
-  # @param: category {"All"=>2413461, "Newspaper"=>1483736, "Images"=>6549, "Books"=>16438} 
-  # @return: A single-key hash in the form of {"More"=> 1483736} 
+  # @author Jeffery
+  # @param category {"All"=>2413461, "Newspaper"=>1483736, "Images"=>6549, "Books"=>16438}
+  # @return A single-key hash in the form of {"More"=> 1483736}
   def self.more_categories_sum(category_counts)
     main_tabs  = ['All', 'Images', 'Audio', 'Videos', 'Sets']
-    blacklist = [ 'Article', 'Music Score', 'Groups', 'Items', 'Websites', 'Research Papers', 'Magazines and Journals', 'Pieces', 'Unknown', 'Interactive', 'Video'] 
+    blacklist = [ 'Article', 'Music Score', 'Groups', 'Items', 'Websites', 'Research Papers', 'Magazines and Journals', 'Pieces', 'Unknown', 'Interactive', 'Video']
+    full_blacklist = blacklist + main_tabs
```

Comment:
I get what you mean, I think this should remain main_tabs though as it is related to the main tabs on the search panel :).